linetools/lists/linelist.py

```diff
@@ -371,7 +564,8 @@ def __getitem__(self,k, tol=1e-3*u.AA):
 
         Returns:
         ----------
-        dict (from row in the data table) or Table (tuple)
+        dict (from row in the data table if only 1 line is found) or QTable (tuple
```

Comment:
I think you wrote the original documentation; here tuple refers to the input 'k'